### PR TITLE
Add redhat_access_lib

### PR DESCRIPTION
**Pull request by ares**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.17
* [ ] 1.16
* [ ] 1.15

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by ares** [comment]
I regenerated this with `./add_gem_package.sh redhat_access_lib scl foreman-plugins-nightly-rhel7` but it only added changelog

File: comps/comps-foreman-plugins-rhel7.xml [hunk 1]
```diff
@@ -24,6 +24,7 @@
       <packagereq type="default">rubygem-newt</packagereq>
       <packagereq type="default">rubygem-openscap</packagereq>
       <packagereq type="default">rubygem-radcli</packagereq>
+      <packagereq type="default">rubygem-redhat_access_lib</packagereq>
```
Comment (ekohl) [inline comps/comps-foreman-plugins-rhel7.xml]: This shouldn't be here.

File: comps/comps-foreman-plugins-rhel7.xml [hunk 2]
```diff
@@ -146,6 +148,7 @@
       <packagereq type="default">rubygem-mysql2-doc</packagereq>
       <packagereq type="default">rubygem-newt-doc</packagereq>
       <packagereq type="default">rubygem-radcli-doc</packagereq>
+      <packagereq type="default">rubygem-redhat_access_lib-doc</packagereq>
```
Comment (ekohl) [inline comps/comps-foreman-plugins-rhel7.xml]: This shouldn't be here

**Comment by ekohl** [comment]
> I regenerated this with ./add_gem_package.sh redhat_access_lib scl foreman-plugins-nightly-rhel7 but it only added changelog

When the package is already present, it bumps it. We should rename it to something that better covers what it does.

**Comment by ekohl** [comment]
http://koji.katello.org/koji/taskinfo?taskID=93023

**Pull request merged**